packages/utils-dom/tests/html.ts

```diff
@@ -28,6 +28,11 @@ test('HTMLDocument.base should return the base url', (t) => {
     t.is(t.context.document.base, 'http://example.com/resources/');
 });
 
+test('HTMLDocument.compatMode should return the compat mode', (t) => {
+    t.is(t.context.document.compatMode, 'CSS1Compat');
```

Comment:
Let's add a test case for a `BackCompat` scenario too.

Reply:
done, I forced push so we could rebase directly from the website once CI passes.